.github/workflows/blackbox-tests.yaml

```diff
@@ -0,0 +1,43 @@
+name: "blackbox-tests"
+
+on:
+  workflow_dispatch:
+
+env:
+  CGO_ENABLED: "0"
+
+jobs:
+  blackbox-tests:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: "Set up Go"
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.18
+
+    - name: "Configure go modules cache"
+      uses: actions/cache@v3
+      with:
+        path: |
+          ~/.cache/go-build
+          ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+
+    - name: "Install dependencies"
+      run: |
+        go mod tidy
+        go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
We probably don't need this anymore right?

Reply:
That's true. ack

Reply:
addressed